File: swift/Utils/ConfigUtils.m
#import "ConfigUtils.h"

#pragma mark - config keys
#define KEY_BACKGROUND @"background"

@implementation ConfigUtils

+(void) saveBackgroundImage:(NSString *)imgName {
    NSUserDefaults * def = [NSUserDefaults standardUserDefaults];
    [def setObject:imgName forKey:KEY_BACKGROUND];
    [def synchronize];
}

+(NSString *) loadBackgroundImage {
    NSUserDefaults * def = [NSUserDefaults standardUserDefaults];
    return [def objectForKey:KEY_BACKGROUND];
}

@end

File: swift/Utils/ConfigUtils.swift
import UIKit

let KEY_BACKGROUND = "background"

class ConfigUtils: NSObject {
    
    class func saveBackgroundImage(imgName: String) {
        let def = UserDefaults.standard()
        def.set(imgName, forKey: KEY_BACKGROUND)
        def.synchronize()
    }
    
    class func loadBackgroundImage() -> String? {
        let def = UserDefaults.standard()
        return def.object(forKey: KEY_BACKGROUND) as? String
    }
}
